tools/istio-iptables/pkg/utils/utils.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import "strings"
+
+func FormatIptablesCommand(commands [][]string) []string {
```

Comment:
This function formats a list of commands, so that should be reflected in the name:
```suggestion
func FormatIptablesCommands(commands [][]string) []string {
```

Reply:
done